### ISSUE TITLE
@/codeforafrica Join our slack

### ISSUE THREAD

**Issue by damolaoladipo** [CodeForAfrica/ui#637]

**Comment by kilemensi** [comment]
When issue are done @damolaoladipo, they should be **closed**... is there a way of automating that?

**Comment by damolaoladipo** [comment]
Yes, We can automate it, using GitHub actions. I'd look it this. @kilemensi

**Comment by kilemensi** [comment]
> Yes, We can automate it, using GitHub actions. I'd look into this. @kilemensi

If the only way is via GitHub Actions @damolaoladipo then lets leave it for now...
